src/network/networkd-network.c

```diff
@@ -1311,6 +1315,90 @@ int config_parse_ignore_carrier_loss(
         return 0;
 }
 
+int config_parse_dhcp_nft_set_context(
+                const char *unit,
+                const char *filename,
+                unsigned line,
+                const char *section,
+                unsigned section_line,
+                const char *lvalue,
+                int ltype,
+                const char *rvalue,
+                void *data,
+                void *userdata) {
+        Network *network = userdata;
+
+        assert(filename);
+        assert(lvalue);
+        assert(rvalue);
+        assert(network);
+
+        return config_parse_nft_set_context(unit, filename, line, section, section_line, lvalue, ltype, rvalue, &network->dhcp_nft_set_context, &network->n_dhcp_nft_set_contexts);
+}
```

Comment:
these wrappers are unnecessary, just use the offset field in the gperf file, see above

Reply:
removed